strawberryfields/backends/bosonicbackend/backend.py

```diff
@@ -184,7 +202,7 @@ def init_circuit(self, prog):
             _New_modes,
         )
 
-        non_gauss_preps = (Bosonic, Catstate, DensityMatrix, Fock, GKP, Ket)
+        non_gauss_preps = (Bosonic, Catstate, DensityMatrix, Fock, GKP, Ket, _New_modes)
```

Comment:
what does `_New_modes` do?

Reply:
_New_modes is what gets called when you use New() in a circuit. It's best to include it here, I think, since this is where we instantiate all the modes, and because it could potentially be used to create a non-Gaussian state.

Reply:
> It's best to include it here, I think, since this is where we instantiate all the modes, and because it could potentially be used to create a non-Gaussian state.

Worth leaving a small comment on that in the code.

Reply:
Done